aday/Challenges-Course/week7/cv/backend/.env.example

```diff
@@ -0,0 +1 @@
+DDBB_URL=mongodb+srv://adayarana:Grasolpa_8gc@cluster0.mmige.mongodb.net/skylab
```

Comment:
You must not leave your url api in public.

Reply:
True, en el .env.example solo poner: DDBB_URL=YOUR_URL o algo así